esp-hal/src/uart.rs

```diff
@@ -1794,7 +1789,6 @@ where
     /// # let mut uart = Uart::new(
     /// #     peripherals.UART0,
     /// #     Config::default())?;
-    ///
     /// uart.apply_config(&Config::default().with_baudrate(19_200))?;
     /// # Ok(())
```

Comment:
You can put a single `#` to stop rendering the row, but still place some visual separation into the comment itself.